src/kata2.booleanToWord.js

```diff
@@ -1,5 +1,9 @@
 const booleanToWord = (boolean) => {
-
+    if (boolean === false || boolean === 0 || boolean === "" || boolean === null || boolean === undefined || boolean === NaN) {
```

Comment:
A boolean is either `true` or `false`. This is different to `0`, `""`, `null`, `undefined` or `NaN`, which are all considered **falsy** values, so you've probably gone a little further than needed here. One other thing is you can use a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) to make this more concise:

```js
return boolean ? 'Yes' : 'No'
```